packages/material-ui/src/Divider/Divider.test.js

```diff
@@ -36,6 +44,55 @@ describe('<Divider />', () => {
     expect(wrapper.hasClass(classes.flexItem)).to.equal(true);
   });
 
+  describe('prop: children', () => {
+    it('should render with the children', () => {
+      const { container } = render(<Divider>content</Divider>);
+      expect(container.querySelectorAll('span').length).to.equal(1);
+    });
+
+    it('should set the default text class', () => {
+      const wrapper = shallow(<Divider orientation={'vertical'}>content</Divider>);
+      expect(wrapper.hasClass(classes.text)).to.equal(true);
+    });
+
+    describe('prop: orientation', () => {
+      it('should set the textVertical class', () => {
+        const wrapper = shallow(<Divider orientation={'vertical'}>content</Divider>);
+        expect(wrapper.hasClass(classes.textVertical)).to.equal(true);
+      });
+    });
+
+    describe('prop: textAlign', () => {
+      it('should set the textRight class', () => {
+        const wrapper = shallow(<Divider textAlign={'right'}>content</Divider>);
+        expect(wrapper.hasClass(classes.textRight)).to.equal(true);
+      });
+
+      it('should set the textLeft class', () => {
+        const wrapper = shallow(<Divider textAlign={'left'}>content</Divider>);
+        expect(wrapper.hasClass(classes.textLeft)).to.equal(true);
+      });
+
+      it('should not set the textRight class', () => {
```

Comment:
```suggestion
      it('should not set the textRight class if orientation="vertical"', () => {
```
Let's update the test description to match exactly what we are trying to test